tapir/coop/templates/coop/shareowner_list.html

```diff
@@ -29,31 +29,37 @@ <h5>{% translate "Cooperative Members" %}</h5>
-                    <form action="" method="get" class="form member-filter-form">
-                        <div class="member-filter-form-filters">{% bootstrap_form filter.form %}</div>
-                        <div class="member-filter-form-buttons">
-                            <button class="filter-button {% tapir_button_link %}">
-                                <span class="material-icons">filter_alt</span>{% translate 'Filter' %}
-                            </button>
-                            <a class="{% tapir_button_link %}"
-                               href="{% url 'coop:shareowner_list' %}">
-                                <span class="material-icons">clear</span>
-                                {% translate "Clear all filters" %}
-                            </a>
-                            <div class="ms-2">
-                                {% blocktranslate %}
-                                    Filtered {{ filtered_member_count }} of {{ total_member_count }}
-                                {% endblocktranslate %}
-                            </div>
-                        </div>
-                    </form>
-                </li>
+                <ul class="list-group list-group-flush" id="share_owner_table">
+                    <div class="collapse" id="collapseFilter">
+                        <li class="list-group-item">
+                            <form action="" method="get" class="form member-filter-form">
+                                <div class="member-filter-form-filters">{% bootstrap_form filter.form %}</div>
+                                <div class="member-filter-form-buttons">
+                                    <button class="filter-button {% tapir_button_link %}">
+                                        <span class="material-icons">filter_alt</span>{% translate 'Filter' %}
+                                    </button>
+                                    <a class="{% tapir_button_link %}"
+                                    href="{% url 'coop:shareowner_list' %}">
+                                        <span class="material-icons">clear</span>
+                                        {% translate "Clear all filters" %}
+                                    </a>
+                                    <div class="ms-2">
+                                        {% blocktranslate %}
+                                            Filtered {{ filtered_member_count }} of {{ total_member_count }}
+                                        {% endblocktranslate %}
+                                    </div>
+                                </div>
+                            </form>
+                        </li>
+                    </div>
             {% endif %}
-            <li class="list-group-item">{% render_table table %}</li>
-        </ul>
-    </div>
+                    <li class="list-group-item">{% render_table table %}</li>
+                </ul>
 {% endblock content %}
```

Comment:
I think I asked about it previously. Could you please add screenshots in case when the UI is changing? Otherwise it's tricky to review.

Reply:
https://github.com/user-attachments/assets/b5be8772-b0dd-4551-96ae-390e695008d2

Reply:
Thanks for the video :clap: How about putting the new button on the same line as `Export` button? I think it's better to keep the buttons next to each other. Like so:

![image](https://github.com/user-attachments/assets/29823b34-7f58-45a1-981f-3446e8cdc054)